src/main/java/org/jenkinsci/maven/plugins/hpi/hosting/PreferLTS.java

```diff
@@ -0,0 +1,42 @@
+package org.jenkinsci.maven.plugins.hpi.hosting;
+
+import org.apache.maven.project.MavenProject;
+import org.jenkinsci.maven.plugins.hpi.hosting.VerificationMessage.Severity;
+
+public class PreferLTS implements VerificationRule {
+	@Override
+	public VerificationMessage validate(MavenProject project) {
+		MavenProject jenkinsParent = findPluginParentPom(project);
+		if (jenkinsParent == null) {
+			return new VerificationMessage(
+					"Your plugin doesn't seem to have the Jenkins parent pom in its POM hierarchy. "
+							+ "Jenkins plugins generally inherit from the Jenkins parent POM.",
+					Severity.WARNING);
+		}
+		String jenkinsVersion = jenkinsParent.getVersion();
+
+		// In 2.x plugin-pom became a standalone project https://github.com/jenkinsci/plugin-pom
+		// In that case, the Jenkins version is specified in the property "jenkins.version".
+		if(jenkinsVersion.startsWith("2."))
+		{
+			jenkinsVersion = (String) project.getProperties().get("jenkins.version");
+		}
+		if (jenkinsVersion.split("\\.").length != 3) {
+			return new VerificationMessage("Your plugin does not seem to have a LTS Jenkins release. In general, "
+					+ "it's preferable to use an LTS version as parent version.",
```

Comment:
My preference is to depend on weekly releases *unless* there is a specific API that you need from the LTS. Otherwise you risk breaking the plugin for anyone on the `2.(N+1)` as the back-ports to `2.N.M` may have originated from `2.(N+2)` or newer.

So I would be against this warning. Better would be to either check if there exists a corresponding LTS line or have a configuration option to enable this specific check